includes/rest.php

```diff
@@ -50,3 +74,69 @@ function check_endpoint( $request ) {
 
 	return rest_ensure_response( $secure_version_exists );
 }
+
+/**
+ * Prepares the parameters for the fix-related endpoints from the provided WP_REST_Request
+ *
+ * @param WP_REST_Request $request Request object.
+ *
+ * @return array
+ */
+function prepare_fix_params( WP_REST_Request $request ): array {
```

Comment:
I suggest using the REST APIs schema for this.

Reply:
This method does nothing more than prepare the params coming from the request for the methods that implement that counting and the fixing of posts with insecure content. How do you think we could use the REST APIs schema for this?